physionet-django/user/templates/user/edit_trainings.html

```diff
@@ -0,0 +1,57 @@
+{% extends "user/settings.html" %}
+
+{% block title %}PhysioNet Trainings{% endblock %}
```

Comment:
{{ SITE_NAME }} ?

Reply:
Nice catch! It's been rebased after debranding PR was merged and I forgot about that.